### PR TITLE
FEATURE: MSF-17743 upgrade custom v2 for rollout brick

### PR DESCRIPTION
**Pull request by tqtu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/398/ : SUCCESS in 4s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/443/ : SUCCESS in 25m 43s

File: lib/gooddata/lcm/actions/synchronize_date_dimension.rb [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# (C) 2019-2020 GoodData Corporation
+require_relative 'base_action'
+
+module GoodData
+  module LCM2
+    class SynchronizeDateDimension < BaseAction
+      DESCRIPTION = 'Synchronize Date Dimension'
+      DATE_DIMENSION_CUSTOM_V2 = 'custom_v2'
+      DATE_DIMENSION_OLD = %w[gooddata custom]
```
Comment (danh-ung) [inline lib/gooddata/lcm/actions/synchronize_date_dimension.rb:9]: please define full urn: urn:gooddata:date or urn:custom:date to urn:custom_v2:date

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/399/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/444/ : SUCCESS in 25m 48s

**Comment by tqtu** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/401/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/446/ : SUCCESS in 25m 18s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/402/ : SUCCESS in 4s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/447/ : SUCCESS in 25m 23s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/403/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/448/ : SUCCESS in 24m 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/407/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/449/ : SUCCESS in 23m 04s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/409/ : SUCCESS in 5s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/451/ : SUCCESS in 22m 57s

File: lib/gooddata/lcm/actions/synchronize_date_dimension.rb [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+# (C) 2019-2020 GoodData Corporation
+require_relative 'base_action'
+
+module GoodData
+  module LCM2
+    class SynchronizeDateDimension < BaseAction
```
Comment (danh-ung) [inline lib/gooddata/lcm/actions/synchronize_date_dimension.rb:7]: Please rename to MigrateGdcDateDimension, file name migrate_gdc_date_dimension.rb.
Document: Migrate date dimension urn:gooddata:date or urn:custom:date to urn:custom_v2:date

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/410/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/452/ : SUCCESS in 28m 07s

**Comment by tqtu** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/411/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/453/ : SUCCESS in 25m 35s

**Comment by tqtu** [comment]
ok  to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/412/ : SUCCESS in 3s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/454/ : SUCCESS in 24m 42s

File: lib/gooddata/lcm/actions/synchronize_date_dimension.rb (hunk 2 above)
Comment (tqtu) [reply lib/gooddata/lcm/actions/synchronize_date_dimension.rb]: Yes, I renamed, and fixed all relation issues

**Comment by tqtu** [comment]
close to open PR on develop branch

**Pull request closed without merging**